DispatchOverlayShims.h issue with Sourcery 0.11.0

> ❌  error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/shims/DispatchOverlayShims.h:185:20: error: implicit conversion of C pointer type 'dispatch_data_t _Nonnull' (aka 'struct dispatch_data_s *') to Objective-C pointer type '__swift_shims_dispatch_data_t _Nonnull' (aka 'id') requires a bridged cast

I've tried with both Xcode 9.2 and Xcode 9.3. Reverting to Sourcery 0.10.1 fixes the issue.

--- a/SourcerySwift/Sources/SwiftTemplate.swift
+++ b/SourcerySwift/Sources/SwiftTemplate.swift
@@ -273,7 +273,7 @@ private extension String {
 }
 
 private extension Process {
-    static func runCommand(path: String, arguments: [String], environment: [String: String] = ProcessInfo.processInfo.environment) throws -> ProcessResult {
+    static func runCommand(path: String, arguments: [String], environment: [String: String] = [:]) throws -> ProcessResult {
         let task = Process()
         task.launchPath = path
         task.arguments = arguments

